R/stats.R

```diff
@@ -0,0 +1,91 @@
+#' Summary statistic functions
+#'
+#' `{messydates}` provides extensions to a variety of functions
+#' with R's `{stats}` package to simplify the creation of
+#' summary reports of data with `{skimr}`.
+#' All functions take a `mdate` vector as an input and return a single
+#' statistic.
+#' Note that individual dates are first expanded and resolved
+#' before a vector level statistic is computed
+#' (e.g. `maxmax()` corresponds to the maximum date of the maximum
+#' of the vector resulting from an expanded `{mdate}`)
```

Comment:
This could be clearer. From what I understand, these functions are needed because there are two levels of summary with mdates: at the level of each observation and at the level of the vector of observations. Is that correct?